packages/expo-web-browser/src/WebBrowser.ts

```diff
@@ -317,6 +324,20 @@ export function maybeCompleteAuthSession(
   return { type: 'failed', message: 'Not supported on this platform' };
 }
 
+function _processOptions(options: WebBrowserOpenOptions) {
+  return {
+    ...options,
+    controlsColor:
+      options.controlsColor != null ? processColor(options.controlsColor) : options.controlsColor,
```

Comment:
Looks like the `processColor` already handles `null` and `undefined` pretty well: https://github.com/facebook/react-native/blob/e509f96baf5e523301a5c9567c416508ff20d175/Libraries/StyleSheet/processColor.js#L23-L26